gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -487,449 +418,4 @@ end
-		tData:AddDescriptionLine(
-			TryT(ent.sb_tag.txt),
-			ent.sb_tag.color
-		)
-	end
-
-	-- add hints to the player
-	local hint = ent.TargetIDHint
-
-	if hint and hint.hint then
-		tData:AddDescriptionLine(
-			hint.fmt(ent, hint.hint),
-			COLOR_LGRAY
-		)
-	end
-end
-
--- handle looking ragdolls
-function HUDDrawTargetIDRagdolls(tData)
-	local client = LocalPlayer()
-	local ent = tData:GetEntity()
-	local c_wep = client:GetActiveWeapon()
-
-	-- has to be a ragdoll
-	if not IsValid(ent) or ent:GetClass() ~= "prop_ragdoll" then return end
-
-	-- only show this if the ragdoll has a nick, else it could be a mattress
-	if not CORPSE.GetPlayerNick(ent, false) then return end
-
-	local corpse_found = CORPSE.GetFound(ent, false) or not DetectiveMode()
-	local role_found = corpse_found and ent.search_result and ent.search_result.role
-	local binoculars_useable = IsValid(c_wep) and c_wep:GetClass() == "weapon_ttt_binoculars" or false
-	local role = roles.GetByIndex(role_found and ent.search_result.role or 1)
-
-	-- enable targetID rendering
-	tData:EnableText()
-	tData:EnableOutline(tData:GetEntityDistance() <= 100)
-	tData:SetOutlineColor(COLOR_YELLOW)
-
-	-- add title and subtitle to the focused ent
-	tData:SetTitle(
-		corpse_found and CORPSE.GetPlayerNick(ent, "A Terrorist") or TryT("target_unid"),
-		role_found and COLOR_WHITE or COLOR_YELLOW
-	)
-
-	if tData:GetEntityDistance() <= 100 then
-		if cvDeteOnlyInspect:GetBool() and client:GetBaseRole() ~= ROLE_DETECTIVE then
-			if client:IsActive() and client:IsShopper() and CORPSE.GetCredits(ent, 0) > 0 then
-				tData:SetSubtitle(ParT("corpse_hint_inspect_only_credits", key_params))
-			else
-				tData:SetSubtitle(TryT("corpse_hint_no_inspect"))
-			end
-		elseif cvDeteOnlyConfirm:GetBool() and client:GetBaseRole() ~= ROLE_DETECTIVE then
-			tData:SetSubtitle(ParT("corpse_hint_inspect_only", key_params))
-		else
-			tData:SetSubtitle(ParT("corpse_hint", key_params))
-		end
-	elseif binoculars_useable then
-		tData:SetSubtitle(ParT("corpse_binoculars", {key = Key("+attack", "ATTACK")}))
-	else
-		tData:SetSubtitle(TryT("corpse_too_far_away"))
-	end
-
-	-- add icon to the element
-	tData:AddIcon(
-		role_found and role.iconMaterial or materialCorpse,
-		role_found and role.color or COLOR_YELLOW
-	)
-
-	-- add hints to the corpse
-	local hint = ent.TargetIDHint
-
-	if hint and hint.hint then
-		tData:AddDescriptionLine(
-			hint.fmt(ent, hint.hint),
-			COLOR_LGRAY
-		)
-	end
-
-	-- add info if searched by detectives
-	if ent.search_result and ent.search_result.detective_search and client:IsDetective() then
-		tData:AddDescriptionLine(
-			TryT("corpse_searched_by_detective"),
-			DETECTIVE.ltcolor,
-			{materialDetective}
-		)
-	end
-
-	-- add credits info when corpse has credits
-	if client:IsActive() and client:IsShopper() and CORPSE.GetCredits(ent, 0) > 0 then
-		tData:AddDescriptionLine(
-			TryT("target_credits"),
-			COLOR_YELLOW,
-			{materialCredits}
-		)
-	end
-end
+end
```

Comment:
please do not remove the empty line at the file end.

Reply:
so you have empty lines at the end of each file?